README.md

```diff
@@ -17,7 +17,7 @@ wiki](https://github.com/DA0-DA0/dao-contracts/wiki/DAO-DAO-Contracts-Design).
 | [cw20-stake](contracts/staking/cw20-stake)                                     | A contract for staking cw20 tokens.                    |
 | [cw20-stake-external-rewards](contracts/staking/cw20-stake-external-rewards)   | A contract for providing external stakinig rewards.    |
 | [cw20-stake-reward-distributor](contracts/staking/cw20-stake-external-rewards) | A contract for distributing rewards via stake-cw20.    |
-|                                                                                |                                                        |
+
 | Unaudited contracts                                                            | Description                                            |
 |:-------------------------------------------------------------------------------|:-------------------------------------------------------|
 | [cwd-proposal-multiple](contracts/proposal/cwd-proposal-multiple)              | A proposal module for multiple choice proposals.       |
```

Comment:
**nit**: these probably need to get added here

```suggestion
| [cwd-voting-cw721-staked](contracts/proposal/cwd-voting-cw721-staked)              | A voting power module for staked cw721s       |
| [cwd-voting-native-staked](contracts/proposal/cwd-voting-native-staked)              | A voting power module for staked native tokens that are not used to secure the chain e.g. JUNO on Juno Network.       |
| [cwd-voting-staking-denom-staked](contracts/proposal/cwd-voting-staking-denom-staked)              | A voting power module for staked native tokens. This module allows you to use the chain's stake token as the contract's staking token e.g. staking JUNO in this module on the Juno Network.       |
| [cwd-proposal-multiple](contracts/proposal/cwd-proposal-multiple)              | A proposal module for multiple choice proposals.       |
```

Reply:
great catch. thank you!